tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -38,6 +38,10 @@ runv sed -i 's|^gpgkey.*|gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Offic
 case "${AUTOPKGTEST_REBOOT_MARK:-}" in
 "")
     echo "Testing overriding with CentOS Stream kernel"
+    # Disable all repos except baseos and appstream as not all of them have support for all RHCOS supported architectures
+    runv sed -i 's/enabled\ =\ 1/enabled\ =\ 0/g' /etc/yum.repos.d/cs.repo
+    runv sed -i '/\[baseos\]/,/^ *\[/ s/enabled\ =\ 0/enabled\ =\ 1/' /etc/yum.repos.d/cs.repo
+    runv sed -i '/\[appstream\]/,/^ *\[/ s/enabled\ =\ 0/enabled\ =\ 1/' /etc/yum.repos.d/cs.repo
```

Comment:
oh man. it would be really nice if the work we did in https://github.com/coreos/rpm-ostree/pull/4346 applied to the non-container path too.

Reply:
agreed, that would have been nice for this